service/gcs/main.go

```diff
@@ -15,21 +16,40 @@ import (
 )
 
 func main() {
-	// parse command line parameters and init logger
-	if err := utils.ProcessCommandlineOptions(); err != nil {
-		logrus.Fatalf("%+v", err)
+	logLevel := flag.String("loglevel", "warning", "Logging Level: debug, info, warning, error.")
+	logFile := flag.String("logFile", "", "Logging Target: An optional file name/path. Omit for console output.")
+
+	flag.Usage = func() {
+		fmt.Fprintf(os.Stderr, "\nUsage of %s:\n", os.Args[0])
+		flag.PrintDefaults()
+		fmt.Fprintf(os.Stderr, "Examples:\n")
+		fmt.Fprintf(os.Stderr, "    %s -loglevel=debug -logfile=gcslog.txt (default)\n", os.Args[0])
```

Comment:
-logfile=<anything> will specify to output the logs to a file. If you omit the parameter you will get logs to stdout. So yes its still relevant unless I misunderstood your question.